packages/react-components/react-overflow/src/Overflow.cy.tsx

```diff
@@ -416,7 +416,7 @@ describe('Overflow', () => {
     cy.get(`[${selectors.divider}]`).should('have.length', 1);
   });
 
-  it.only('should remove overflow menu if the last overflowed item can take its place', () => {
+  it('should remove overflow menu if the last overflowed item can take its place', () => {
```

Comment:
Removing `.only` here because it triggers a lint error below and it actually resulted in only this cypress test being run which i'm not sure was intentional or not @ling1726 :

```
  419:3  error  'it.only' is restricted from being used. it.only should only be used during test development

Reply:
whoops, that's definitely not intentional, thx for correcting this